crates/aiken/src/cmd/blueprint/apply.rs

```diff
@@ -26,6 +26,10 @@ pub struct Args {
     /// `plutus.json`), or use the `cbor.serialise` function from the Aiken standard library.
     parameter: Option<String>,
 
+    /// Path to a blueprint file to be used as input.
+    #[clap(short = 'b', long, value_parser)]
+    blueprint_input: Option<PathBuf>,
```

Comment:
not a rust expert, but does this mean `-b  --blueprint-input` will be the two flags for the cli, give `-o --output` is for output, wouldn't it be better to have `-i --input` ? After all subcommands already define the "context" `aiken blueprint apply`

my 2 lovelaces

Reply:
https://github.com/aiken-lang/aiken/commit/7c4e0444234b4138d6251acdccb4d33b3297f9e4